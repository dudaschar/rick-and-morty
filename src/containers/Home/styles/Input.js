import styled from 'styled-components';

import Input from 'src/components/Input/Input';

export const StyledInput = styled(Input)`
  margin: 0;
  margin-right: ${({ theme }) => theme.spacing.small};
  height: 100%;
`;

StyledInput.displayName = 'StyledInput';
