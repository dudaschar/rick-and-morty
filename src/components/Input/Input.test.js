import React from 'react';
import { render } from 'src/testUtils/testUtils';

import Input from './Input';

describe('Input Component', () => {
  it('should render properly', () => {
    const { asFragment } = render(<Input />);

    expect(asFragment()).toMatchSnapshot();
  });
});