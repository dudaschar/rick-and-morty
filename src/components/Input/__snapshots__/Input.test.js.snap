// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component should render properly 1`] = `
<DocumentFragment>
  <input
    class="BaseInput-b9nobq-0 jgnjtU"
    type="text"
  />
</DocumentFragment>
`;
